### ISSUE TITLE
feature/CSS/adding class styling for input box

### ISSUE THREAD

**Issue by williampowell03** [williampowell03/will-p-july24#40]
There is no styling for the input boxes. Currently the input boxes are standard HTML styling which lacks accessibility for the general User.

This styling will need to be added to resize the input boxes and set the font styling
